frontend/src/scenes/settings/project/Proxy.tsx

```diff
@@ -88,23 +87,29 @@ export function Proxy(): JSX.Element {
     ]
 
     return (
-        <div className="space-y-2">
-            <LemonTable
-                loading={proxyRecords.length === 0 && proxyRecordsLoading}
-                columns={columns}
-                dataSource={proxyRecords}
-                expandable={{
-                    expandedRowRender: (record) => <ExpandedRow record={record} />,
-                }}
-            />
-            {formState === 'collapsed' ? (
-                <LemonButton onClick={showForm} type="secondary" icon={<IconPlus />}>
-                    Add domain
-                </LemonButton>
+        <PayGateMini feature={AvailableFeature.HOSTED_REVERSE_PROXY}>
```

Comment:
Should this `isCloudOrDev` check be outside of the paygate? I feel like non-cloud users should see the warning always.

Reply:
Good call, updated